roles/ceph-rgw/tasks/rgw_pool_pgs.yml

```diff
@@ -2,7 +2,23 @@
 - name: create rgw pools if create_pools is defined
```

Comment:
You can remove this task

Reply:
Sorry @guits Im no following your reasoning. Why removing this task? Its the one that creates pools when not containerized. I guess what you want is set a fact depnding on containerized/non conatinerized and then having just one task that uses the fact to execute the command. Am I wrong?